zooplankton.gemspec

```diff
@@ -23,5 +23,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "pry"
   spec.add_development_dependency "rspec", "~> 2.14"
 
-  spec.add_dependency "rails", "~> 4.0"
+  spec.add_dependency "rails", ">= 4.0"
```

Comment:
I think we should break this out to be greater than or equal to `4.0`, but less than `6.0`:
```
spec.add_dependency "rails", [">= 4.0", "< 6.0"]
```
Right? Like a super twiddlewakka.